package.json

```diff
@@ -26,7 +26,7 @@
     "/test",
     "/truffle-config.js"
   ],
-  "author": "Aragon Institution MTU <contact@aragon.one>",
+  "author": "Aragon Association <contact@aragon.one>",
```

Comment:
Should we change this to `contact@aragon.org` or another aragon.org email?

Reply:
Indeed